modules/installation-three-node-cluster.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws/installing-aws-user-infra.adoc [Eventually]
+// * installing/installing_azure/installing-azure-user-infra.adoc [Eventually]
+// * installing/installing_gcp/installing-gcp-user-infra.adoc [Eventually]
+// * installing/installing_gcp/installing-restricted-networks-gcp.adoc [Eventually]
+// * installing/installing_bare_metal/installing-bare-metal.adoc
+// * installing/installing_aws/installing-restricted-networks-aws.adoc [Eventually]
+// * installing/installing_bare_metal/installing-restricted-networks-bare-metal.adoc
+// * installing/installing_vsphere/installing-restricted-networks-vsphere.adoc [Eventually]
+// * installing/installing_vsphere/installing-vsphere.adoc [Eventually]
+// * installing/installing_ibm_z/installing-ibm-z.adoc [Eventually]
+
+[id="installation-three-node-cluster_{context}"]
+= Running a three-node cluster
+
+[IMPORTANT]
+====
+Three-node clusters are currently a Technology Preview feature.
+ifdef::openshift-enterprise,openshift-webscale[]
+Technology Preview features are not supported with Red Hat production service
+level agreements (SLAs), might not be functionally complete, and Red Hat does
+not recommend to use them for production. These features provide early access to
+upcoming product features, enabling customers to test functionality and provide
+feedback during the development process.
+
+See the link:https://access.redhat.com/support/offerings/techpreview/[Red Hat
+Technology Preview features support scope] for more information.
+endif::[]
+====
+
+You can install and run a three-node clusters in {product-title}. This provides smaller, more resource efficient clusters for cluster administrators and developers to use for deployment, development, and testing.
```

Comment:
Either "run a three-node cluster" or "run three-node clusters"

Reply:
Also just wondering whether you can give any more info on exactly what three-node clusters consist of. I assume per the procedure below that it's 3 masters and no workers, but might be helpful to point out here.